cmd/dtbs/dtbs.go

```diff
@@ -0,0 +1,267 @@
+package databaseshortener
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	bn "go-prof-sprint-1/cmd/bindata"
+
+	flw "go-prof-sprint-1/cmd/flwrk"
+
+	"github.com/golang-migrate/migrate"
+	"github.com/golang-migrate/migrate/database/sqlite3"
+	bindata "github.com/golang-migrate/migrate/source/go_bindata"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/pkg/errors"
+)
+
+func NewDB(dbPath string) (*sql.DB, error) {
```

Comment:
CHANGE REQUEST: please add an interface

https://dakaii.medium.com/repository-pattern-in-golang-d22d3fa76d91